components/booking/BookingContactForm.vue

```
@@ -0,0 +1,97 @@
+
+const props = withDefaults(defineProps<{
+  initialState?: Partial<ContactFormState>
+  loading?: boolean
+  submitText?: string
+}>(), {
+  submitText: 'Confirm booking',
+  loading: false
+})
+
+const emit = defineEmits<{
+  (e: 'submit', form: ContactFormState): void
```

ESLint

[error] 'e' is defined but never used. Allowed unused args must match /^_/u.

(no-unused-vars)